### PR TITLE
Container setup

### PR DESCRIPTION
**Pull request by ej2brown**
Able to have multiple pages

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,15 +1,30 @@
 import React from 'react';
```
Comment (thecameraguy) [inline src/index.tsx:1]: Let's clean up the import style to be more consistent: import { blah } from 'blahblah'; - you can assume the interviewer would be looking at your code while reading your resume. The cleaner and more polished the better!!

**Comment by thecameraguy** [comment]
👍 after polishing up the import statements. After that merge away! 🚢

File: src/index.tsx (hunk 1 above)
Comment (ej2brown) [reply src/index.tsx:1]: I don't think it's possible, here's why:
https://stackoverflow.com/questions/54800819/why-does-import-react-from-react-not-work

Comment (thecameraguy) [reply src/index.tsx:1]: Interesting! I learned something new today. Merge away!!

**Pull request merged**